mnelab/mainwindow.py

```diff
@@ -222,6 +222,17 @@ def __init__(self, model):
         self.actions["events_from_annotations"] = tools_menu.addAction(
             "Create events from annotations", self.events_from_annotations
         )
+        tools_menu.addSeparator()
+        self.nirs_menu = tools_menu.addMenu("NIRS")
```

Comment:
We don't need an attribute for this menu because we never access it except here in `__init__`. So just use `nirs_menu` instead of `self.nirs_menu`.